### ISSUE TITLE
Find a way to handle / update / cache network information (network id, scores)

### ISSUE THREAD

**Issue by aaronlevin** [omarieclaire/common#9]
We don't have a good story for where the network information is stored:

* when rendering, it's helpful to know what network each node belongs to (for example, so we can draw circles around the sub-networks)
* certain actions can impact the network structure (e.g. destroyer, giver, clicking, etc.), so how or where do we persist that information (is clicking an action?)? 
* If we persist the network as a separate object, how do we propagate new information to the nodes who need to know what network they belong to for rendering?

Basically: need to figure out how to get the network information out of the main app and tracked into firebase, _or_ decide explicitly not to do that and handle it in the browser (as it is now).
